stream_alert_cli/terraform_generate.py

```diff
@@ -629,6 +683,27 @@ def generate_athena(config):
         'prefix': config['global']['account']['prefix']
     }
 
+    if not enable_metrics:
+        return athena_dict
+
+    # Check to see if there are any metrics configured for the athena function
+    current_metrics = metrics.MetricLogger.get_available_metrics()
+    if metrics.ATHENA_PROCESSOR not in current_metrics:
+        return athena_dict
+
+    metric_prefix = 'AthenaRefresh'
+    filter_pattern_idx, filter_value_idx = 0, 1
+
+    # Add filters for the cluster and aggregate
+    # Use a list of strings that represnt the following comma separated values:
+    #   <filter_name>,<filter_pattern>,<value>
+    filters = ['{},{},{}'.format('{}-{}'.format(metric_prefix, metric),
+                                 settings[filter_pattern_idx],
+                                 settings[filter_value_idx])
+               for metric, settings in current_metrics[metrics.ATHENA_PROCESSOR].items()]
```

Comment:
Nit: `.iteritems()`

Reply:
@austinbyers I had suggested using `.items()` since `.iteritems()` is not a py3 thing. However you raise a good point that I didn't think of when I originally suggested this to @jacknagz . `.iteritems()` returns a generator while `.items()` returns a copy of the items as tuples.. Since that's the case we're probably better off using `.iteritems()` for the time being (so I take back what I previously said @jacknagz sorry about this !! :/)